### DISCUSSION TITLE
Question about using variables in Conditions blocks

### DISCUSSION THREAD

**Discussion by jlopez994** [AutomaApp/automa#1339] [Q&A] [answered]
Hi,

I'm new using Automa and I can't find info about using variables into conditions in order to create paths with them.

Any suggestion?

**This is what I'm trying now but doesn't work.**

<img width="562" alt="image" src="https://github.com/AutomaApp/automa/assets/1363023/8e9a6787-8477-40b1-872a-c95217aef235">

Thanks in advance.

**Comment by SuperTempAccount** [comment, marked as answer]
Hey,

Edit: Seems that you forgot to wrap the variable in double brackets like this `{{variables.action}}`

The latest version of Automa uses the syntax `variables.action` instead of `variable@action` (although I believe both are still working).
Try turning on debugging mode to see what the condition is evaluating to
<img width="501" alt="image" src="https://github.com/AutomaApp/automa/assets/83566267/031f0d3e-276c-4f00-b82d-4a2da6113ca8">

**Comment by jlopez994** [comment]
Yep, that was the problem. Thank you!
